jobs/v4/job_search_autocomplete_job_title_test.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import job_search_autocomplete_job_title
+
+PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
+TENANT_ID = "b603d325-3fb5-4979-8994-eba4ecf726f4"
```

Comment:
Does this text expect that there is a `tenant` already created in the project?
If so, how can I create the tenant? Use case: I might want to run this test against my own project.

Reply:
you can create a tenant using the sample code `job_search_create_tenant.py`.

TENANT_ID expect that project has already created that tenant.
I wanted to reduce the complexity by removing creating tenant in setup and deleting tenant in teardown. 

Downside: If anyone deletes the tenant, tests will fail.